### PR TITLE
Add integration for ZeroTrustAnalyticsPlatform

### PR DESCRIPTION
**Pull request by vimpadawan**
Add ZTAP Alert incident type

Add custom layout/playbook for ZTAP Alerts

Sync attachments

Add configurable open/close sync for remote incidents

Add classifiers/mappers for bidirectional syncing
The following are synced both ways:
* Comments (as Notes)
* Close status

The following data are pulled directly from ZTAP
* Alert ID
* Assigned Organization
* Escalate Organization
* Priority
* Product Status
* Source Organization
* Trigger Count (Trigger Event Count)

The following are processed
* Fields for trigger events (ZTAPExtractFields)
* Triggers (Trigger Events) for display in a dynamic view (ZTAPParseFields)
* Links to external sites related to the alert (ZTAPParseLinks)

The following are used for internal use by XSOAR
* Input Tag (synced for ZTAPBuildTimeline)

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Add new integration for syncing ZTAP alerts.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by daryakoval** [comment]
@vimpadawan Hi, Please fill the Contribution Registration form, and the I will start a code-review.

**Comment by vimpadawan** [comment]
> @varangamin Hi, Please fill the [Contribution Registration form](https://forms.gle/XDfxU4E61ZwEESSMA) and sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=15390) .

Forms should be submitted now

**Comment by daryakoval** [comment]
@vimpadawan Let’s schedule a demo call. Update me when you are available.

**Comment by vimpadawan** [comment]
@daryakoval My availability for the next few days
11/01 11AM - 5PM 
11/02 9AM - 10 AM, 1 PM - 5PM
11/03 9AM - 5PM
All times are CDT

**Comment by daryakoval** [comment]
> @daryakoval My availability for the next few days 11/01 11AM - 5PM 11/02 9AM - 10 AM, 1 PM - 5PM 11/03 9AM - 5PM All times are CDT

Sent an invitation 11/03 9AM.

File: Packs/ZeroTrustAnalyticsPlatform/Tests/id_set.json [hunk 1]
```diff
@@ -0,0 +1,832 @@
+{
```
Comment (daryakoval) [inline Packs/ZeroTrustAnalyticsPlatform/Tests/id_set.json:1]: Please remove this file.

**Comment by evisochek** [comment]
@vimpadawan Please consider the following:
1)Adding task that will check if the integration is available
2)Replace the entity enrichment playbook for V3
3)Change the task type of manually review the incident to manual task and not conditional

File: Packs/ZeroTrustAnalyticsPlatform/Playbooks/playbook-ZTAP_Alert_README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+This playbok is triggered by fetching escalated ZTAP Alerts.
+The playbook fetches newly escalated alerts.
+Then, the playbook performs enrichment on the incident's indicators.
+Lastly, it adds comments/logs as Evidence.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* Calculate Severity - Generic v2
```
Comment (evisochek) [inline Packs/ZeroTrustAnalyticsPlatform/Playbooks/playbook-ZTAP_Alert_README.md:10]: Please generate new README file after the playbook modifications

**Review by evisochek — APPROVED** [review]

File: Packs/ZeroTrustAnalyticsPlatform/Playbooks/playbook-ZTAP_Alert_README.md (hunk 2 above)
Comment (daryakoval) [reply Packs/ZeroTrustAnalyticsPlatform/Playbooks/playbook-ZTAP_Alert_README.md:10]: And add new picture.

**Comment by daryakoval** [comment]
@vimpadawan Please see our last comment: https://github.com/demisto/content/pull/15390#discussion_r742648778

**Comment by vimpadawan** [comment]
@daryakoval thought I pushed this fix Friday, but accidentally went to the wrong remote

**Review by daryakoval — APPROVED** [review]

**Pull request merged**